### PR TITLE
Add CONTRIBUTING document

### PR DESCRIPTION
**Pull request by zgjs**
Hey team, this MR adds the CONTRIBUTING document for the repo. We used [contributing.md](https://contributing.md/) to help create the template and added information specific to our project.

 :speech_balloon: Please consider this a draft and suggest any additional changes that you think would be helpful before we merge it into `main`. 

Part of #25

### CODE REVIEW

**Comment by magictrick-dev** [comment]
We should probably add a link to this into the README. Otherwise LGTM.

**Comment by zgjs** [comment]
> We should probably add a link to this into the README. Otherwise LGTM. - @magictrick-dev 

Good catch. We were thinking that it since it is a special file, it would be pinned to the repo info skinny or somewhere prominent, and we could also link to it in the PR template, but we should also link to it in the README.

(And we should also add the PR template for the second task in #25. I'm going to convert this MR to draft until we have those two changes and then I'll ask for re-review when it's undrafted.)

File: docs/CONTRIBUTING.md
Comment (magictrick-dev) [inline docs/CONTRIBUTING.md:1]: Overall, I think it looks fine.

**Review by magictrick-dev — COMMENTED** [review]
I will await further examination from other e-board members to determine if this is a good fit. The contributing document is concise and seems to check all the boxes on our requirements.

**Review by magictrick-dev — APPROVED** [review]
LGTM!

**Comment by Jacobcon3315** [comment]
Reading through the document, it looks good! I have no comments on anything that should be added or removed.

**Comment by Migs** [comment]
The documment looks good. Should we add a coding guidelines section (i.e. format, comments, casing, etc), or have that be a separate file?

**Comment by zgjs** [comment]
> The documment looks good. Should we add a coding guidelines section (i.e. format, comments, casing, etc), or have that be a separate file?

I think @ExplosiveEggshells was working on coding guidelines in a separate document, but we could update this one with a link to it.

**Review by Migs — APPROVED** [review]
LGTM. We can add the documentation later on.

**Review by Jacobcon3315 — APPROVED** [review]

**Review by marcelinegallegos — APPROVED** [review]
Looks good to me! No further comments

**Pull request merged**